src/reflect/value.go

```diff
@@ -863,10 +982,41 @@ func alloc(size uintptr, layout unsafe.Pointer) unsafe.Pointer
 //go:linkname sliceAppend runtime.sliceAppend
 func sliceAppend(srcBuf, elemsBuf unsafe.Pointer, srcLen, srcCap, elemsLen uintptr, elemSize uintptr) (unsafe.Pointer, uintptr, uintptr)
 
+//go:linkname hashmapStringGet runtime.hashmapStringGet
+func hashmapStringGet(m unsafe.Pointer, key string, value unsafe.Pointer, valueSize uintptr) bool
```

Comment:
I'm not sure this will work in LLVM 14 because the type of the `m` parameter is a different pointer type.